src/api/schema.ts

```diff
@@ -194,18 +218,28 @@ export const lists = {
         },
       }),
       creditPeriodWithBuyer: text(),
-      policyCurrencyCode: text(),
+      policyCurrencyCode: text({
+        db: { nativeType: 'VarChar(1000)' },
+      }),
       totalMonthsOfCover: integer(),
       totalSalesToBuyer: integer(),
       maximumBuyerWillOwe: integer(),
-      goodsOrServicesDescription: text(),
+      goodsOrServicesDescription: text({
+        db: { nativeType: 'VarChar(1000)' },
+      }),
       finalDestinationCountryCode: text(),
     },
     access: allowAll,
   },
   ExporterBusiness: list({
     fields: {
-      company: relationship({ ref: 'ExporterCompany' }),
+      application: relationship({ ref: 'Application' }),
+      goodsOrServicesSupplied: text({
+        db: { nativeType: 'VarChar(1000)' },
+      }),
+      yearsExporting: integer(),
+      employeesUK: integer(),
+      employeesInternational: integer(),
```

Comment:
Could we add `total` prefix to the last 3 fields so `totalYearsExporting`, `totalEmployeesUK` etc?